CacheTagsBundle/Listener/ResponseListener.php

```diff
@@ -11,7 +11,7 @@
 use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
 use Symfony\Component\HttpKernel\HttpKernelInterface;
 
-class Response
+class ResponseListener
```

Comment:
I suggested to https://github.com/sensiolabs/SensioFrameworkExtraBundle/blob/master/EventListener/HttpCacheListener.php